tests/Makefile

```diff
@@ -112,6 +104,7 @@ $(BINARIES): %$(EXEXT): %.c $(DEP)
 	echo "  CC    $<"
 	$(CC) $(LOCAL_CFLAGS) $(CFLAGS) $<	$(LOCAL_LDFLAGS) $(LDFLAGS) -o $@
 
+$(filter test_suite_psa_crypto%, $(BINARIES)): psa_helpers.function
```

Comment:
Do I need to do anything else so that the tests can find `psa_helpers.function` when building for on-target testing (`embedded_xxx` targets)?

Reply:
Yes, I do. https://github.com/ARMmbed/mbed-crypto/pull/132/commits/1d10257d215fc2ea366e7b6f15b532e2f40504b9